File: websocket client/SiftGridCustomer/SiftGridCustomer/pkg/Debug/ModularWebservicePkg/Code/wwwroot/app/notes/Notes.ts
export class Notes {

    public NoteId;
    public TenantId;
    public LocationId;
    public CreatedDateTime;
    public CreatedBy;
    public ModifiedDateTime;
    public ModifiedBy;
    public  NoteJson;
    public constructor(NoteId: String, TenantId: String, LocationId: String, CreatedDateTime: String,
        CreatedBy: String, ModifiedDateTime: String, ModifiedBy: String, NoteJson: String)
    {

        this.NoteId = NoteId;
        this.TenantId = TenantId;
        this.LocationId = LocationId;
        this.CreatedDateTime = CreatedDateTime;
        this.CreatedBy = CreatedBy;
        this.ModifiedDateTime = ModifiedDateTime;
        this.ModifiedBy = ModifiedBy;
        this.NoteJson = NoteJson;
    }

}

File: websocket client/SiftGridCustomer/ServiceInterfaces/Notes.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace NotesApi.Model
{
    public class Notes
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.None)]
        public string NoteId { set; get; }
        public string DiagramId { set; get; }
        public string TenantId { set; get; }
        public string TenantName { set; get; }

        public string LocationId { set; get; }
        public string CreatedDateTime { set; get; }
        public string CreatedBy { set; get; }
        public string ModifiedDateTime { set; get; }
        public string ModifiedBy { set; get; }
        public string NoteJson { set; get; }

        public Notes()
        {

        }
    }
}
